ingestion_tools/scripts/importers/visualization_config.py

```diff
@@ -190,11 +219,19 @@ def _create_config(self, alignment_metadata_path: str) -> dict[str, Any]:
 
         for _, info in annotation_layer_info.items():
             args = {**info["args"], "resolution": resolution}
-
-            if info["shape"] == "SegmentationMask":
+            has_mesh = self._has_mesh(args["source_path"])
+            shape = info["shape"]
+            if shape == "SegmentationMask":
+                if has_mesh:
+                    print(
```

Comment:
I wouldn't be inclined to hide the mesh here, I think I'd just hide the layer instead. The user probably won't know how to turn back on the mesh.

I get the concern is that it spawns in with overlapping meshes though. But honestly I think we should just leave it, because if someone setup `is_visualization_default` in the config with overlapping meshes, then they just need to fix the config. They might actually want to show both the seg and the oriented meshes by default and we'd be preventing that.

I think this is one we'll have to confirm with CZI/CZII

Reply:
Alternatively, the config could store to hide the mesh explicitly

Reply:
The problem is that if we remove the layer, we loose the segmentation in the 2D views, that might be interesting. As the mesh in the segmentation layer is disabled, there is no overlapping. However, if you think it's better to not see the segmentation for those, we can remove the layer, but I feel we are loosing a piece of information if we do. What do you think?

Reply:
Yeah I'm really not sure. You could even reduce the mesh opacity or make it a silhoutte so you could see the vectors
![image](https://github.com/user-attachments/assets/6929b588-7910-465d-8297-62763a23bb6c)

Compare to normal
![image](https://github.com/user-attachments/assets/2f83bba6-24cb-41ce-aa50-aff93654963c)

but I can't quite tell if the vector based representation is something that is really important or more like a left over from when it was the only was to visualise it